Misc/Adam.cpp

```diff
@@ -3354,3 +3354,24 @@ int AdamInit()
 
 	return 1;
 }
+
+void WriteAlteredConfigVarsToIni()
+{
+	const char* alteredConfigVars[] = {
+		"projmetroen",
```

Comment:
Same as `itemtimelock` below: this changes the user's defaults for new projects.